lib/phoenix_ecto/html.ex

```diff
@@ -278,8 +308,8 @@ if Code.ensure_loaded?(Phoenix.HTML) do
     end
   end
 
-  if Code.ensure_loaded?(Ecto.DateTime) do
-    defimpl Phoenix.HTML.Safe, for: [Ecto.Time, Ecto.Date, Ecto.DateTime] do
+  if Code.ensure_loaded?(DateTime) do
+    defimpl Phoenix.HTML.Safe, for: [Time, Date, DateTime] do
```

Comment:
Those should rather be moved to phoenix_html. I will work on this.

Reply:
It is already in phoenix_html. So we can just remove this as long as we bump the ecto requirement.